index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo"><a href="#">My Logo</a></li>
+                <li><img src="images/Menu.svg" alt="menu image" class="menu-image"></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="flex-container">
+    <section class="greet-section"> 
+        <img src="images/background.svg" alt="background image" class="banner">
+        <h1>I’m Mariane Glad to see you!</h1>
+    
+        <p class="info-section">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        <p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] As this is an invitation to connect, it's only reasonable that it is a link. It should also use a header tag. Heading tags indicate headings on a webpage by using code to tell a web browser how to display content. That's why and how they organize your content into a format that's easy to read. In addition to general structure and readability, heading tags help improve accessibility for people who can't easily read screens. 
- [ ] Refer to the [design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to add the necessary stylings. Pay attention to the `font-styles/font-size, colours, icon-sizes`. 

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/120834357-c120bd00-c517-11eb-8e67-b93f2a62624c.png) | ![image](https://user-images.githubusercontent.com/49621279/120834413-d564ba00-c517-11eb-82f4-13c178e55920.png)

Reply:
This issue is not addressed. Kindly make the suggested changes by the previous TSE. :pray: